README.md

```diff
@@ -1 +1,222 @@
+- **This projects follows all Microverse Linters**
+- **This project fetches data from a Space API**
+- **This project displays fetched data of Space dynamically in the DOM**
+- **This project pushes reservations to an API**
+- **This project fetches reservations from an API**
+- **This project has syntax with ES6 format.**
+- **This project makes use of React-Redux.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to copy the link of this Repository.
+
+
+
+```sh
+ https://github.com/josuerushanika/space-travelers.git
+```
+<br>
+
+### Setup
+
+Clone this repository to your desired directory using the command: 
+
+
+```sh
+  cd your-folder
+  git clone https://github.com/josuerushanika/space-travelers.git 
+```
+
+<br>
+
+
+### Usage
+
+Kindly go ahead and give this project a massage but do not forget to give a detailed description in your pull request.
+
+
+Install the required dependencies using the following command:
+
+```sh
+  npm install
+```
+
+Run the server using the following command:
+
+```sh
+  npm start
+```
+-
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORs -->
+
+## 👥 Authors <a name="authors"></a>
+### 👥 Frempong
+
+- GitHub: [@frempongdev](https://github.com/frempongdev)
+- Twitter: [@knobaddy](https://twitter.com/knobaddy)
+- LinkedIn: [Samuel knower Amankwaa-Frempong](https://www.linkedin.com/in/samuel-knower-amankwaa-frempong-356802256/)
+
+
+👤 **Josue Rushanika**
+
+- GitHub: https://github.com/josuerushanika
+- Twitter: https://twitter.com/josuerushanika
+- LinkedIn: https://www.linkedin.com/in/josue-rushanika-336049b3/
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Ticket Issue Page]**
+- [ ] **[Animations for Page]**
+- [ ] **[Server Storage]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project kindly support by suggesting changed by forking and pull requests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank  everyone who inspired me to learn programming. Regards to the whole Microverse team, Peers, and family supporting. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] the link to the MIT license is not working as was intended in [the template ](https://github.com/microverseinc/readme-template)provided by Microverse, please have a look and change this.

Reply:
Link has been fixed.